client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { PerformanceMetrics } from 'calypso/performance-profiler/types/performance-metrics';
+import {
+	metricsNames,
+	metricsTresholds,
+	mapThresholdsToStatus,
+} from 'calypso/performance-profiler/utils/metrics';
+import { MetricScale } from '../metric-scale';
+import { MetricTabBar } from '../metric-tab-bar';
+import './style.scss';
+import { StatusIndicator } from '../status-indicator';
+
+export const CoreWebVitalsDisplay = ( props: PerformanceMetrics ) => {
+	const translate = useTranslate();
+	const [ activeTab, setActiveTab ] = useState< string >( 'lcp' );
+
+	const { name: metricName, displayName } = metricsNames[ activeTab as keyof typeof metricsNames ];
+	const value = props[ activeTab as keyof PerformanceMetrics ];
+	const valuation = mapThresholdsToStatus( activeTab as keyof typeof metricsTresholds, value );
+
+	const displayValuation = ( valuation: string ) => {
+		switch ( valuation ) {
+			case 'good':
+				return 'is good';
+			case 'needsImprovement':
+				return 'needs improvement';
+			case 'bad':
+				return 'is bad';
+			default:
+				return 'unknown';
+		}
+	};
+
+	const { good, needsImprovement } = metricsTresholds[ activeTab as keyof typeof metricsTresholds ];
+	const formatUnit = ( value: number ) => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return +( value / 1000 ).toFixed( 2 );
+		}
+
+		return value;
+	};
+
+	const displayUnit = () => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return 'seconds';
+		}
+
+		if ( [ 'inp' ].includes( activeTab ) ) {
+			return 'milliseconds';
+		}
+
+		return '';
```

Comment:
Same as above.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/93411/commits/29f95b09e425e41053145bd30207ee7cc561c8b7